projects/packages/search/src/dashboard/components/record-meter/bar-chart.jsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Chart from 'chart.js/auto';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export class BarChart extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.canvasRef = React.createRef();
+	}
+
+	componentDidMount() {
+		this.myChart = new Chart( this.canvasRef.current, {
+			type: 'bar',
+			options: {
+				borderRadius: 100,
+				borderSkipped: 'middle',
+				indexAxis: 'y',
+				maintainAspectRatio: false,
+				aspectRatio: 1.3,
+				scales: {
+					x: {
+						stacked: true,
+						grid: {
+							display: false,
+							drawBorder: false,
+						},
+						ticks: {
+							display: false,
+						},
+					},
+					y: {
+						stacked: true,
+						grid: {
+							display: false,
+							drawBorder: false,
+						},
+						ticks: {
+							display: false,
+						},
+					},
+				},
+
+				plugins: {
+					title: {
+						display: false,
+					},
+					subtitle: {
+						display: false,
+					},
+					legend: {
+						display: false,
+
+						labels: {
+							filter: function ( legendItem ) {
+								return ! legendItem.text.includes( 'Remaining' );
```

Comment:
Will this still work in non-English languages?

Reply:
Hmm .. good point. Likely not! Let me see what I can do to fix that up 😄